src/pages/communication-page/cat-d/communication.cat-d.page.html

```diff
@@ -0,0 +1,54 @@
+<ion-header>
+  <ion-navbar>
+    <ion-title>{{ 'communication.title' | translate }} - {{pageState.candidateUntitledName$ | async}}</ion-title>
+    <ion-buttons end>
+      <end-test-link category="C"></end-test-link>
```

Comment:
we should get the category from state here and pass it in

Reply:
done